Signal/src/ViewControllers/ConversationView/Cells/OWSGenericAttachmentView.m

```diff
@@ -116,25 +117,37 @@ - (void)createContents
     [contentView autoPinEdgeToSuperviewEdge:ALEdgeTop withInset:self.vMargin];
     [contentView autoPinEdgeToSuperviewEdge:ALEdgeBottom withInset:self.vMargin];
 
+    OWSBezierPathView *iconCircleView = [OWSBezierPathView new];
+    UIColor *iconColor
+        = (self.isIncoming ? [UIColor colorWithRGBHex:0x9e9e9e] : [self foregroundColorWithOpacity:0.15f]);
+    iconCircleView.configureShapeLayerBlock = ^(CAShapeLayer *_Nonnull layer, CGRect bounds) {
+        CGFloat radius = MIN(bounds.size.width, bounds.size.height) * 0.5f;
+        layer.path = [UIBezierPath bezierPathWithRoundedRect:bounds cornerRadius:radius].CGPath;
+        layer.fillColor = iconColor.CGColor;
+    };
+    [contentView addSubview:iconCircleView];
+    [iconCircleView autoPinEdgeToSuperviewEdge:ALEdgeLeft withInset:self.iconHMargin];
+    [iconCircleView autoVCenterInSuperview];
+    [iconCircleView autoSetDimension:ALDimensionWidth toSize:self.iconSize];
+    [iconCircleView autoSetDimension:ALDimensionHeight toSize:self.iconSize];
+    [iconCircleView setContentHuggingHigh];
```

Comment:
No biggie, but since we're explicitly setting the width and height, does setting contentHugging have any affect?

Reply:
Done.